kotlinx-coroutines-core/common/src/flow/Flow.kt

```diff
@@ -199,7 +199,7 @@ public abstract class AbstractFlow<T> : Flow<T> {
 
     @InternalCoroutinesApi
     public final override suspend fun collect(collector: FlowCollector<T>) {
-        collectSafely(SafeCollector(collector, collectContext = coroutineContext))
+        collectSafely(SafeCollector(collector, coroutineContext))
```

Comment:
`try { .. } finally { ... }` should be here, too.

Reply:
good catch